src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -115,20 +139,22 @@ final case class RequestSession[F[_]: Monad](
   def flatMap[R, R1](r: F[R], f: R => F[R1]): F[R1] = r.flatMap(f)
 }
 
-class GenericClient[F[_]: Monad: Comonad, _](
+class GenericClient[F[_]: Monad: Comonad, S](
     applicationName: String,
     baseUri: String =
       Option(System.getenv("COGNITE_BASE_URL")).getOrElse("https://api.cognitedata.com")
 )(
     implicit auth: Auth,
-    sttpBackend: SttpBackend[F, _]
+    sttpBackend: SttpBackend[F, S]
 ) {
   import GenericClient._
   val uri: Uri = try {
     uri"$baseUri"
   } catch {
     case _: Throwable =>
-      throw new IllegalArgumentException("Unable to parse URI. Please check URI syntax.")
+      throw new IllegalArgumentException(
+        s"Unable to parse URI ${baseUri}. Please check URI syntax."
```

Comment:
I'd suggest putting the URI in quotes here

Reply:
Databricks doesn't like quotes, so I'd rather not, but `"Unable to parse baseUri = ${baseUri} as a valid URI."` maybe?

Reply:
Agh, that's annoying. Oh well, that looks okay to me :)